build.gradle

```diff
@@ -27,6 +27,8 @@ buildscript {
             "mockk"             : "1.9.3",
             "robolectric"       : "4.3.1",
             "startup"           : "1.0.0-alpha02",
+            "composeCompiler"   : "1.0.0-alpha03",
```

Comment:
Compose compiler v1.0.0-alpha04 is now out too. I'm not an expert on this, but I think the compose runtime and the compiler are meant to share the same version number.

Reply:
I know, but for some reason they haven't uploaded it to their [Maven ](https://maven.google.com/web/index.html?q=compose#androidx.compose:compose-compiler) repo yet.